app/models/comment.rb

```diff
@@ -13,6 +13,9 @@ class Comment < ApplicationRecord
   self.table_name = 'comments'
   self.primary_key = 'cid'
 
+  COMMENT_FILTER = "\n@@$$%% Trimmed Content @@$$%%\n"
```

Comment:
Interesting and clever! What about putting it into an HTML comment as well - like `<!-- $$%% TRIMMED CONTENT %%$$ -->` so that it can be parsed out with an HTML filter if need be? I dunno, just seems potentially extra helpful in the future...

Reply:
This way would be great. Putting this in html comment didn't come to my mind. 👍 🎉